src/collapse/perf.rs

```diff
@@ -245,12 +301,17 @@ impl PerfState {
                 }
             }
 
-            // TODO: show_inline
+            // --inline flag
+            if self.opt.show_inline && can_un_inline(module) {
+                let un_inlined = self.un_inline(pc, module)?;
```

Comment:
I think we _probably_ don't want to exit the entire parsing just because we couldn't resolve a particular symbol. Instead, we probably just want to print an error and move on. That would also let us not propagate around this custom error value everywhere. Thoughts?

Reply:
How do you want to handle errors as a whole for this program? The main function originally returned io::Result. I could switch back to that. Would you rather not have a custom error type for the program to be able to return other types of errors? I guess anything that's not an IO error can just be printed as you're suggesting for this.

Reply:
Yeah, I think that's the behavior we want. Specifically, we want `collapse-perf` to try to produce output to the greatest extent that it can, and failing to resolve a particular address probably shouldn't count. Maybe we should fall back to `[unknown 0x....]` in that case? I believe the only errors that would _truly_ stop progress would be I/O errors, so yeah, I think we can stick with that.

Reply:
Perfect, that makes sense.

Reply:
Now, this _might_ mean that we want to keep track of warnings (instead just dumping them with `eprintln!`). Perhaps it's time to introduce [`log`](https://docs.rs/log/) into inferno?

Reply:
I added log and replaced all uses of `eprintln!`. I'll let you choose a logging implementation.